lang/English/modules/settings.php

```diff
@@ -45,7 +45,7 @@
 define('feed_url', "Feed URL");
 define('feed_url_info', "GrayCube.com is sharing a LGSL feed on the URL:<br><b>http://www.greycube.co.uk/lgsl/feed/lgsl_files/lgsl_feed.php</b>");
 define('charset', "Character Encoding");
-define('charset_info', "UTF8, ISO, ASCII, etc... Left it blank to use ISO encoding.");
+define('charset_info', "UTF8, ISO, ASCII, etc... Leave it blank to use ISO encoding.");
```

Comment:
I don't even know if we still need this parameter actually. as we defined the encoding from lang files now (I think it was not like that long long time ago, will check sourceforge for that => yeah it was not like that on really old version). Maybe some inputs from own3mall on the topic will enlighten us more.